ansible/roles/ansible-horizon-plugin/tasks/install_ubuntu.yml

```diff
@@ -15,3 +15,17 @@
     - python-workloadmgrclient
   notify:
       - restart webserver
+  when: PYTHON_VERSION=="python2"
+  
+- name: install python3-tvault-horizon-plugin and python3-workloadmgrclient deb packages when using python3
+  apt:
+     update_cache: yes
+     name: "{{ item }}={{ TVAULT_PACKAGE_VERSION }}"
+     state: present
+     allow_unauthenticated: yes
+  with_items:
+    - python3-tvault-horizon-plugin
+    - python3-workloadmgrclient
```

Comment:
workloadmgr should get installed before tvault-horizon-plugin to avod pacakge denedancies.

Reply:
Requested changes done